lib/eclipse/include/opm/parser/eclipse/EclipseState/Tables/AquanconTable.hpp

```diff
@@ -0,0 +1,48 @@
+/*
+  Copyright (C) 2015 Statoil ASA.
+  2016 IRIS AS
+
+  This file is part of the Open Porous Media project (OPM).
+
+  OPM is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  OPM is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with OPM.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#ifndef OPM_PARSER_AQUANCON_TABLE_HPP
+#define	OPM_PARSER_AQUANCON_TABLE_HPP
+
+#include "SimpleTable.hpp"
+
+namespace Opm {
+
+    class DeckItem;
+
+    class AquanconTable : public SimpleTable {
```

Comment:
The `AQUANCON` keyword does not implement a table; I might grudgingly accept the (private) use of `SimpleTable` inheritance and the `TableManager` for internalizing the `AQUTAB` keyword, but not for the `AQUANCON` keyword - this is *not* a table.

I don't know the full details of this, but you should implement something like an `AquiferConfig` class and that should among other things include the internalized content of the `AQUANCON` kewyord.